src/main/java/org/openelisglobal/common/services/ResultService.java

```diff
@@ -0,0 +1,42 @@
+package org.openelisglobal.common.services;
```

Comment:
I don’t see a reason for this class to exist and in general we should avoid static variables for services like this static, but managed by spring (some pre-existing classes have this as an artifact from the migration to Spring, but it's use should not be encouraged).